container/nginx/njs/badbits.js

```diff
@@ -0,0 +1,34 @@
+const fs = require("fs");
+const crypto = require("crypto");
+
+const badbits = {};
+const data = JSON.parse(fs.readFileSync("/etc/nginx/denylist.json"));
+data.forEach((entry) => (badbits[entry.anchor] = true));
```

Comment:
Could also use Sets here:

```js
const badbits = new Set()
badbits.add(entry.anchor)
badbits.has(hashedCID)
```

Reply:
That makes a lot of sense. Sadly, it will yield `2023/03/05 13:57:11 [error] 41#41: *1 js exception: ReferenceError: "Set" is not defined`. I had initially tried with a `Map`.

Reply:
ah dang it's too old a version then :|